app/javascript/application.js

```
@@ -0,0 +1,32 @@
+    families: ['Roboto:300,400,500']
+  }
+});
+
+document.addEventListener('turbo:load', () => {
+  const csrfToken = document.querySelector('meta[name="csrf-token"]')?.getAttribute('content');
+
+  if (csrfToken) {
+    Vue.http.headers.common['X-CSRF-TOKEN'] = csrfToken;
+    axios.headers['X-CSRF-Token'] = csrfToken;
+  }
+  return config;
```

ESLint

[error] 'config' is not defined.

(no-undef)